packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -1217,6 +1231,8 @@ const String _pluginCmakefileTemplate = r'''
 # Generated file, do not edit.
 #
 
+# clang-format off
```

Comment:
clang format shouldn't be running on CMake files.

Reply:
ahh!